### ISSUE TITLE
Command exists in other env

### ISSUE THREAD

**Issue by woutdenolf** [pyenv/pyenv#1568]
I have a system command "/usr/bin/h5cc" which I want to use in one environment (lets call it env1). However I can't because it happens to be installed in another environment (lets call it env2) so I get the message:

```
pyenv: h5cc: command not found

The `h5cc' command exists in these Python versions:
  miniconda3-latest/envs/env2
```

Is there a way for my to force "h5cc" to be "/usr/bin/h5cc" in env1?

**Comment by woutdenolf** [comment]
Note that "h5cc" is call but something else so I can use the full path to execute the command

**Comment by joshfriend** [comment]
anaconda likes to provide its own versions of very common shell utilities, and keeps adding more all the time. pyenv keeps an ["exclude list"](https://github.com/pyenv/pyenv/blame/master/pyenv.d/rehash/conda.d/default.list) of things from anaconda's `bin/` that should not be shimmed, but it is very much a losing battle trying to keep up with them.

**Comment by ireneu** [comment]
This behavior is unexpected from the user point of view, in which global commands in $PATH are assumed to be always available.

Could pyenv default to non-pyenv versions when they exist?

Otherwise, is there an applicable workaround for users?

**Comment by native-api** [comment]
Fixed by cdecf1453555b22d22a8f88a9bcce71244855042
